component/object.jsonnet

```diff
@@ -40,7 +40,7 @@ local objectDumper =
   local image = params.images.object_dumper;
   local pod = backup.PreBackupPod(
     'object-dumper',
-    '%s:%s' % [ image.image, image.tag ],
+    '%s/%s:%s' % [ image.registry, image.image, image.tag ],
```

Comment:
This will explode if anyone uses `images.object_dumper.image` to overwrite the registry

Reply:
I'm making it a breaking change anyways.